proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -377,6 +378,55 @@ message FieldOptions {
   int32 jstype = 4;
 }
 
+// FieldLiteral represents a field and its value. It can be used to model descriptor options.
+message FieldLiteral {
+  // When is_extension is true, name is the fully-qualified name of the
+  // extension. Otherwise, it is the simple name of the field.
+  string name = 1;
+  // The unique field number associated with the field.
+  int32 tag = 2;
+  // Indicates whether this is an extension or not.
+  bool is_extension = 3;
+  // The field's value.
+  LiteralValue value = 4;
+}
+
+// MessageLiteral represents an option value for a type that is a message.
+message MessageLiteral {
+  // Map of field names to value. Extension field names will be in the form:
+  // [fully.qualified.extension.Name].
+  repeated FieldLiteral fields = 1;
```

Comment:
I suppose Any will have a "synthetic" field with the type URL here?

https://protobuf.com/docs/language-spec#any-messages

Reply:
Great catch!